### PR TITLE
Revert "Merge branch 'master' into tests"

### PR DESCRIPTION
**Pull request by davidyuk**
From #44 I can't get why my refactoring was reverted there, I'm planning to use the same code in another project, so proposing to revert them back here.

### CODE REVIEW

File: src/popup/router/components/SidebarMenu.vue [hunk 1]
```diff
@@ -92,27 +92,20 @@ export default {
   mixins: [removeAccountMixin],
   components: { Close, Arrow, UserAvatar },
   computed: mapGetters(['account', 'activeAccountName']),
-  data: () => ({ showSettingsDropdown: false, showTokensDropdown: false, balances: null }),
-  watch: {
-    showTokensDropdown(val) {
```
Comment (mradkov) [inline src/popup/router/components/SidebarMenu.vue:97]: This should be present for the FT functionality I think.

Comment (davidyuk) [reply src/popup/router/components/SidebarMenu.vue:97]: Well, I'm proposing to introduce it in the same time as fungible tokens support.

**Comment by davidyuk** [comment]
For some reason it breaks menu clicking in e2e tests, I will investigate it later

**Comment by davidyuk** [comment]
This one is too old, I don't think that I finish this fix

**Pull request closed without merging**